media/css/sandstone/fira-fonts.less

```diff
@@ -0,0 +1,70 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@font-face {
+    font-family: 'Fira Sans';
+    src: url('/media/fonts/FiraSans-Regular.eot?#iefix') format('embedded-opentype'),
+         url('/media/fonts/FiraSans-Regular.woff2') format('woff2'),
```

Comment:
I think this warrants some further investigation. Most instances of bullet-proof `@font-face` syntax recommend putting `.eot` first. Not quite sure the reason, but I believe old IE's may have had issues with different ordering. I'm going to leave this as-is for now and we can always revisit when we're more confident.